index.js

```diff
@@ -104,7 +104,20 @@ module.exports = (input, opts) => {
 
 	if (runPublish) {
 		tasks.add({
-			title: 'Publishing package',
+			title: 'Publishing package using Yarn',
+			enabled: () => opts.yarn === true,
+			skip: () => {
+				if (pkg.private) {
+					return 'Private package: not publishing to Yarn.';
+				}
+			},
+			task: () => exec('yarn', ['publish', '--new-version', input]).catch(err => {
```

Comment:
You also need to support the `tag` option, like done here: https://github.com/ErisDS/np/blob/dbef1d544c7a8f3ccd0e97c43a4c47df0fa58aec/lib/publish.js#L10-L12

Reply:
And it doesn't really make sense to run `--new-version` here, as we already call `yarn version --new-version` earlier: https://github.com/ErisDS/np/blob/dbef1d544c7a8f3ccd0e97c43a4c47df0fa58aec/index.js#L96

Reply:
`yarn publish` wants you to provide an argument for a new version to publish or you'll be prompted for it, so I think this is the only way to avoid the prompt.

Reply:
As @gpoole explained, this isn't unnecessary duplication, it's required by yarn. I don't like it either.

I have added the `--tag` argument back as requested.